champions/src/main/java/me/mykindos/betterpvp/champions/listeners/ArrowListener.java

```diff
@@ -77,6 +83,7 @@ public void onArrowDamage(DamageEvent event) {
                 float dmgMultiplier = arrows.get(arrow);
                 event.setDamage(event.getDamage() * dmgMultiplier);
                 arrows.remove(arrow);
+                arrow.remove();
```

Comment:
Are you sure this won't cause an exception? The arrow might be removed by the game upon hitting an entity, so this may cause an exception by the entity not being loaded. Just double check this

Reply:
can check if the arrow still exists before removing it, but from my testing no error messages happened

Reply:
Just add the piece of code to make sure